src/feature/dirauth/bwauth.c

```diff
@@ -315,6 +316,53 @@ dirserv_read_measured_bandwidths(const char *from_file,
   return rv;
 }
 
+/**
+ * Compute the digest of the bandwidth file used for the vote
+ * and store it in <b>digest_bw_file_out</b>.
+ * Returns -1 on error, 0 otherwise.
+ * It is implemented separated from dirserv_read_measured_bandwidths,
+ * since it should be refactor in the future.
+ */
+int
+bwauth_bw_file_digest255(uint8_t *digest_bw_file_out, const char *from_file,
```

Comment:
This function (and other things in this patch) say "255", but it computes a 256-bit digest.  Probably it should say 256 instead?

Reply:
yes, don't know why i put 255 :)